Include/abstract.h

```diff
@@ -14,9 +14,12 @@ extern "C" {
 
    Print an object 'o' on file 'fp'.  Returns -1 on error. The flags argument
    is used to enable certain printing options. The only option currently
-   supported is Py_Print_RAW.
-
-   (What should be said about Py_Print_RAW?). */
+   supported is Py_PRINT_RAW. By default, PyObject_Print "stringifies" 'o' by
+   calling its __repr__ method. However, if Py_PRINT_RAW is specified
+   in 'flags' the object is "stringified" for printing by invoking its __str__
+   method. If Py_PRINT_RAW is specified and the object does not have a
+   __str__ method, only then will it's __repr__ method be called to perform
+   "stringification". */
```

Comment:
Strike the scare quotes, although not a verb per the OED it is a term of art within the domain:

```suggestion
   supported is Py_PRINT_RAW. By default, PyObject_Print stringifies 'o' by
   calling its __repr__ method. However, if Py_PRINT_RAW is specified
   in 'flags' the object is stringified for printing by invoking its __str__
   method. If Py_PRINT_RAW is specified and the object does not have a
   __str__ method, only then will it's __repr__ method be called to perform
   stringification. */
```

A